src/redismodule.h

```diff
@@ -830,6 +830,14 @@ typedef enum {
     REDISMODULE_ACL_LOG_CHANNEL /* Channel authorization failure */
 } RedisModuleACLLogEntryReason;
 
+typedef enum {
+    REDISMODULE_CONFIG_TYPE_STRING = 0,
+    REDISMODULE_CONFIG_TYPE_ENUM,
+    REDISMODULE_CONFIG_TYPE_NUMERIC,
+    REDISMODULE_CONFIG_TYPE_BOOL,
+    REDISMODULE_CONFIG_TYPE_UNKNOWN
```

Comment:
`REDISMODULE_CONFIG_TYPE_UNKNOWN` will change it's value if new types are added in the future.

Can it have a fixed value such as `-1` or `INT_MAX` instead?